tips/TIP-0048/openapi3-core.yaml

```diff
@@ -3480,19 +3473,20 @@ components:
         - blockState
 
     TransactionMetadataResponse:
-      description: Returns the metadata of a given block.
+      description: Returns the metadata of a given transaction.
       properties:
         transactionId:
           type: string
-          description: The identifier of the block. Hex-encoded with 0x prefix.
+          description: The identifier of the transaction. Hex-encoded with 0x prefix.
         transactionState:
           type: string
           enum:
             - pending
+            - accepted
             - confirmed
             - finalized
             - failed
-          description: If 'pending', the block contains not yet included transaction. If 'confirmed' means transaction is confirmed with the first level of knowledge. If 'finalized' means transaction is included and cannot be reverted anymore. If 'failed' means transaction is not successfully issued due to failure reason.
+          description: If 'pending', the transaction is not included yet. If 'accepted', the transaction is included. If 'confirmed' means transaction is included and its included block is confirmed. If 'finalized' means transaction is included, its included block is finalized and cannot be reverted anymore. If 'failed' means transaction is not successfully issued due to failure reason.
```

Comment:
> If 'failed' means transaction is not successfully issued due to failure reason.
What failure reason can this be?

Reply:
The TransactionFailureReason below.

Reply:
Ok, but is `is not successfully issued` correct then? Or is it successfully issued, but has a problem(TransactionFailureReason)?

Reply:
Yes, it was issued successfully but has a problem, then we keep the failure reason in the metadata.